src/main/resources/interface-specification.yml

```diff
@@ -641,6 +669,68 @@ components:
       schema:
         type: string
   schemas:
+    RiskAnalysisFormConfigResponse:
+      type: object
+      properties:
+        version: 
+          type: string
+        questions: 
+          type: array
+          items:
+            $ref: '#/components/schemas/FormConfigQuestionResponse'
+      required:
+        - version
+        - questions 
+    FormConfigQuestionResponse:
+      type: object
+      properties:
```

Comment:
these fields should also be in the response:
- type (string, mandatory)
- defaultValue (string, mandatory)
- options (object with structure in the json, optional)

Reply:
these are additional fields that are in the json configuration

Reply:
defaultValue is a String and a List on json file, but also is a Boolean type or a String type